.ci_support/check.py

```
@@ -0,0 +1,16 @@
+import tomlkit
+
+
+if __name__ == "__main__":
+    with open("pyproject.toml", "r") as f:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)